doc/api/http.md

```diff
@@ -1479,6 +1513,8 @@ const server = http.createServer((req, res) => {
 }).listen(3000);
 ```
 
+Socket argument can be an instance of {stream.Duplex}, a superclass of Socket.
```

Comment:
Shouldn't you change `* {net.Socket}` above to `* {stream.Duplex}`? This is a property, there are no arguments!